MapleLib/WzLib/WzFile.cs

```diff
@@ -206,14 +206,14 @@ internal bool ParseMainWzDirectory(out string parseErrorMessage, bool lazyParse
             }
             WzBinaryReader reader = new WzBinaryReader(File.Open(this.path, FileMode.Open, FileAccess.Read, FileShare.Read), WzIv);
 
-            this.Header = new WzHeader
-            {
-                Ident = reader.ReadString(4),
-                FSize = reader.ReadUInt64(),
-                FStart = reader.ReadUInt32(),
-                Copyright = reader.ReadNullTerminatedString()
-            };
-            reader.ReadBytes((int)(Header.FStart - reader.BaseStream.Position));
+            this.Header = new WzHeader();
+            this.Header.Ident = reader.ReadString(4);
+            this.Header.FSize = reader.ReadUInt64();
+            this.Header.FStart = reader.ReadUInt32();
+            this.Header.Copyright = reader.ReadString((int)(Header.FStart - 17U));
+
+            reader.ReadBytes(1);
+            reader.ReadBytes((int)(Header.FStart - (ulong)reader.BaseStream.Position));
```

Comment:
From what I see the proper way would be just to not do these 2 byte reads and instead move the `reader.BaseStream.Position` to `Header.FStart`

Reply:
as much as I'd like to do that, that would break the header upon saving the file